src/main/java/com/yahoo/bullet/rest/WebSocketConfig.java

```diff
@@ -37,4 +41,9 @@ public void configureMessageBroker(MessageBrokerRegistry registry) {
         registry.enableSimpleBroker(clientDestination);
         registry.setUserDestinationPrefix(clientDestinationPrefix);
     }
+
+    @Override
+    public void configureWebSocketTransport(WebSocketTransportRegistration registration) {
+        registration.setSendBufferSizeLimit(sessionBufferSizeKB * 1024);
```

Comment:
I believe there are more things we need to or should allow configuring:
https://stackoverflow.com/questions/26543904/stomp-over-websocket-the-send-buffer-size-exceeded-the-allowed-limit?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
Added all of 3.